doc-site/docs/tutorials/notarized-tokens.md

```diff
@@ -0,0 +1,165 @@
+- **Notary-Controlled Oversight** – A designated **notary** approves and submits every transaction, ensuring compliance and control.
+- **Selective Disclosure** – Owners can prove token ownership by selectively revealing transaction details when needed.
+
+This tutorial will guide you through issuing, transferring, and verifying tokens using Paladin’s notarization model.
+
+---
+
+## Prerequisites
+
+Before starting, ensure you have:
+
+1. Completed the [Private Storage Tutorial](./private-storage.md).
+2. A **running Paladin network** with at least three nodes (**Node1, Node2, and Node3**).
+
+---
+
+## Overview
+
+This tutorial will cover:
+
+1. **Deploying a Noto Token** – Creating a “cash token” with a designated notary.
+2. **Minting Tokens** – Issuing new tokens into circulation.
+3. **Transferring Tokens** – Simulating payments by moving tokens between nodes.
+
+💡 **The complete example code is available in the [Paladin example repository](https://github.com/LF-Decentralized-Trust-labs/paladin/tree/main/example/notarized-tokens).**
+
+---
+
+## Step 1: Deploy a Noto Token
+
+First, create a **Noto Factory** instance and deploy a new token. **Node1** will act as the **notary**, responsible for approving and submitting all transactions related to this token. Additionally, **Node1 will be the initial recipient of the minted tokens**.
+
+```typescript
+logger.log("Step 1: Deploying a Noto cash token...");
+const notoFactory = new NotoFactory(paladinClientNode1, "noto");
+const cashToken = await notoFactory.newNoto(verifierNode1, {
+notary: verifierNode1,// The notary overseeing ALL token transactions
+notaryMode: "basic",// The notary mode
+});
+if (!cashToken) {
+logger.error("Failed to deploy the Noto cash token!");
+return false;
+}
+logger.log("Noto cash token deployed successfully!");
+```
+
+### Why the Notary Role Matters
+The **notary** is more than just a minting authority—it plays a fundamental role in the **Noto token model**:
+
+- **Approves and submits all token transactions** to the network.
+- **Maintains full visibility** over all token movements.
+- **Ensures transaction integrity and compliance** with predefined rules.
+
+By designating a notary, every transaction must be verified and approved, ensuring controlled and auditable token transfers.
+
+---
+
+## Step 2: Mint Tokens
+
+With the token contract deployed, let’s **mint** an initial supply of tokens for Node1. This simulates creating new “cash” in the system.
+
+```typescript
+logger.log("Step 2: Minting 2000 units of cash to Node1...");
+const mintReceipt = await cashToken.mint(verifierNode1, {
+  to: verifierNode1,
+  amount: 2000,
+  data: "0x",
+});
+if (!mintReceipt) {
+  logger.error("Failed to mint cash tokens!");
+  return false;
+}
+logger.log("Successfully minted 2000 units of cash to Node1!");
+```
+
+### What Happens Here?
+
+1. **Node1 submits a minting request** to the notary.
```

Comment:
In this case Node1 _is_ the notary... so it will be receiving and validating its own request (may or may not need to spell this out?)